src/marketing/prex-type-mostaddcart.md

```diff
@@ -0,0 +1,31 @@
+---
+ee_only: true
+title: Most added to cart
+---
+
+The _Most added to cart_ recommendation type recommends items most added to a cart by counting the number of sessions where an add-to-cart action occurred within the last seven days.
```

Comment:
I would move the "number of sessions" detail to the "how it works" section. it's not essential to the overview of the rec type, and makes it a mouthful. same with others where we count sessions